templates/CRM/Admin/Form/Setting/Miscellaneous.tpl

```diff
@@ -73,9 +73,26 @@
             </td>
           </tr>
         {/foreach}
-
+        <tr class="crm-miscellaneous-form-block-recentItemsMaxCount">
```

Comment:
@seamuslee001 @pradpnayak  What do `recentItems*` have to do with `remote_profile_submissions_allowed`?